docker/recorder/src/main/java/org/arquillian/cube/docker/impl/client/utils/NumberConversion.java

```diff
@@ -6,13 +6,15 @@
 public class NumberConversion {
 
     public static String humanReadableByteCount(Long bytes, boolean decimal) {
-        int unit = decimal ? 1000 : 1024;
         if (bytes == null) bytes = 0L;
-        if (bytes < unit) return bytes + " B";
-        int exp = (int) (Math.log(bytes) / Math.log(unit));
+        String sign = bytes < 0 ? "-" : "";
+        Long absBytes = Math.abs(bytes);
+        int unit = decimal ? 1000 : 1024;
+        if (absBytes < unit) return sign + absBytes + " B";
```

Comment:
can you do an expanded if? `if(){}` just to not skip that this if have a `return`